docs/usage/faq.md

```diff
@@ -41,6 +41,44 @@ If you're self hosting Renovate, use the latest release if possible.
 
 ## What if I need to .. ?
 
+### Tell Renovate to ask for approval before opening a PR
+
+The normal behavior of Renovate is to open a PR right away whenever there's an update.
+But maybe you want Renovate to ask for approval first, before opening the PR.
+Use the "Dependency Dashboard approval" workflow to get updates for certain packages or certain types of updates only after you give approval via the Dependency Dashboard.
+
+The basic idea is that you create a new `packageRule` and describe what kind of package, or type of updates you want to approve beforehand.
+
+Say you want to manually approve all major `npm` type updates:
+
+```json
+{
+  "packageRules": [
+    {
+      "matchUpdateTypes": ["major"],
+      "matchManagers": ["npm"],
+      "dependencyDashboardApproval": true
+    }
+  ]
+}
+```
+
+Say you want to manually approve all major Jest updates:
+
+```json
+{
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["^jest"],
+      "matchUpdateTypes": ["major"],
+      "dependencyDashboardApproval": true
+    }
+  ]
+}
+```
```

Comment:
Would this work now that we group major `ts-jest` and `jest` packages together?

Or should I pick a different package name to be sure?

Reply:
It would only pick packages starting with `jest`, so doesn't match `ts-jest` 🤗